index.html

```diff
@@ -1128,6 +1128,12 @@ <h3>Workflows and Exchanges</h3>
       <table class="simple api-summary-table"
         data-api-path="/workflows /workflows/{localWorkflowId} /workflows/{localWorkflowId}/exchanges /workflows/{localWorkflowId}/exchanges/{localExchangeId}"></table>
 
+      <p>
+In the workflows and exchanges APIs, a "local" ID refers to an ID that is local to a service instance.
+In other words, an `exchangeId` or `workflowId` refers to a fully qualified URL,
+while a `localExchangeId` or `localWorkflowId` refers to a fragment of the URL.
```

Comment:
I'd recommend using the word "portion" so as not to confuse it with a "fragment" that refers to what follows the `#` sign in some URLs, which isn't what we mean here.

```suggestion
while a `localExchangeId` or `localWorkflowId` refers to a portion of the URL.
```